### DISCUSSION TITLE
Spotfire Mods and Integration Links

### DISCUSSION THREAD

**Discussion by KenACollins** [TIBCOSoftware/spotfire-mods#90] [Q&A] [answered]
The sample code [in the Spotfire Mods repository](https://github.com/TIBCOSoftware/spotfire-mods/tree/master/examples) relies on Google Charts which might be a nice way to get started but I would like to see a more advanced example that uses an information link to a database.

**Comment by mrylande** [comment, marked as answer]
Typically a visualization mod can be connected to any data (embedded, in-database, real-time etc.). As this is more related to configuring and using the mod in a Spotfire analysis rather than developing the mod it is not covered by the examples in the Mods GitHub repo. See [Mod data views](https://tibcosoftware.github.io/spotfire-mods/docs/using-the-api/mod-data-views/) for how the axes declared in the mod's manifest is mapped to data in Spotfire.

**Comment by ayh20** [comment]
The Mod is purely about the way the data is visualised, ie a chart via Google Charts, a complex visualization via D3 etc. All data is passed to your Mod in the same way that data is mapped to a standard Spotfire visualization, ie imported into a DXP, from a Streaming data source, a DB via in information link etc
